kubetest2-gke/deployer/deployer.go

```diff
@@ -42,7 +42,6 @@ var GitTag string
 
 const (
 	defaultFirewallRuleAllow = "tcp:22,tcp:80,tcp:8080,tcp:30000-32767,udp:30000-32767"
-	defaultImage             = "cos"
 	defaultWindowsImage      = WindowsImageTypeLTSC
```

Comment:
Shall we also remove the `defaultWindowsImage`, for consistency reason?

Reply:
sounds good, followed up with second commit to remove windows image defaulting.